src/tests/functional/plugin/shared/include/snippets/exp.hpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2022 Intel Corporation
```

Comment:
Already 2024 year))))))

Reply:
Omg this PR is 2 years old already! 🤣 
Time flies...